features/steps/user_initialisation_steps.py

```diff
@@ -0,0 +1,65 @@
+# pylint: disable=E0102, E0602
+from behave import *
+from selenium.webdriver.support.ui import Select
+
+from features.fixtures import get_chrome_browser
+from features.steps.helpers import sign_in
+from users.factories import AdminUserFactory
+from users.factories import UserFactory
+
+
+@given("an admin exists in database")
+def step_impl(context):
+    context.admin = AdminUserFactory()
+
+
+@when("the user signs up")
```

Comment:
`signs up` => `signs in`.

Reply:
signs up = creates account,
signs in = logs is,
which case is implemented here?

Reply:
The focus is sign up, since that's the feature intended for testing here. That's why factory-based creation is used here as a placeholder for proper code. I just have the user signed in afterwards as confirmation of successful sign up procedure.

Reply:
Ok, I thought its signing in (loggin in).